### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chineduG**
Corrections of misspelled words.

 a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,3 +291,4 @@ Licensed under [MIT](./LICENSE.md)
 
 [ABI stability guide]: https://nodejs.org/en/docs/guides/abi-stability/
 [Node-API support matrix]: https://nodejs.org/dist/latest/docs/api/n-api.html#n_api_n_api_version_matrix
+
```
Comment (Trott) [inline README.md:294]:
```suggestion
````

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ If you have a local clone, you can update it by running:
 ```shell
 git branch -m master main
 git fetch origin
-git branch -u origin/main main
+git branch -u origin/main-main
```
Comment (Trott) [inline README.md:9]:
```suggestion
git branch -u origin/main main
```

File: README.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ APIs exposed by node-addon-api are generally used to create and
 manipulate JavaScript values. Concepts and operations generally map
 to ideas specified in the **ECMA262 Language Specification**.
 
-The [Node-API Resource](https://nodejs.github.io/node-addon-examples/) offers an
+The [Node-API Resource](https://nodejs.github.io/node-addon-examples/) offers an
```
Comment (Trott) [inline README.md:59]: This change replaces a non-breaking space (ASCII 160) with a regular space (ASCII 32). LGTM.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**